README.md

```diff
@@ -25,6 +29,47 @@ For example (doesn't work yet):
 openfisca-us-data cps generate 2019 cps.csv.gz
 ```
 
+### Scripting
+```
+my_acs = ACS()
```

Comment:
We actually don't need to instantiate the classes - `ACS.generate(2016)` will work, and `ACS` in place of `my_acs` everwhere.

Reply:
fancy! replacing them now...